src/pages/profile.tsx

```diff
@@ -363,17 +365,37 @@ function matchOutput({
     match.whitePlayerOne.name,
     match.whitePlayerTwo?.name,
   ].filter(notEmpty);
-  return (
-    <span class="text-sm">
-      On{" "}
-      {match.createdAt.toLocaleString("en-US", {
-        day: "numeric",
-        month: "long",
-      })}
-      , <span class="font-bold">{whiteTeam.join(" & ")}</span> faced off against{" "}
-      <span class="font-bold">{blackTeam.join(" & ")}</span> and{" "}
-      {result === RESULT.DRAW ? "tied" : result === RESULT.WIN ? "won" : "lost"}{" "}
-      with {match.scoreDiff} points.
-    </span>
-  );
+
+  const firstSide = MatchStatistics.getPlayersTeam(match, userId)
```

Comment:
```
const userTeam = MatchStatistics.getPlayersTeam(match, userId)
const firstTeam = userTeam === 'black' ?  blackTeam : whiteTeam;
const lastTeam = userTeam === 'black' ? whiteTeam : blackTeam;
```
then you won't have to duplicate the jsx

Reply:
fixefaxed